main.go

```diff
@@ -1,107 +1 @@
-package main
 
-import (
-	"database/sql"
-	"fmt"
-	"log"
-	"os"
-	
-
-	_ "github.com/mattn/go-sqlite3"
-)
-
-func openDB(dbfile string) (*sql.DB) {
-	db, err := sql.Open("sqlite3", dbfile)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return(db)
```

Comment:
this formatting is strange and not sure if gofmt would have fixed it
`return db`